tests/vmi_multus_test.go

```diff
@@ -787,13 +787,21 @@ var _ = Describe("[Serial]SRIOV", func() {
 		})
 
 		It("[test_id:3985]should create a virtual machine with sriov interface with custom MAC address", func() {
+			mac := "de:ad:00:00:be:ef"
 			vmi := getSriovVmi([]string{"sriov"})
-			vmi.Spec.Domain.Devices.Interfaces[1].MacAddress = "de:ad:00:00:be:ef"
+			vmi.Spec.Domain.Devices.Interfaces[1].MacAddress = mac
+
 			startVmi(vmi)
 			waitVmi(vmi)
 
+			updatedVmi, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
```

Comment:
nit: You can overwrite the original `vmi`. Creating another variable hints that you want to do something with the old one, but you do not.

Same with the other occurrences.

Reply:
Done.